### ISSUE TITLE
Talkback reads bogos numbers of contacts in phone and messaging app 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#145]
```
What steps will reproduce the problem?
1. Search for contacts in phone or messaging app
2. Try to select one of the contact's numbers to which the call or msg is 
directed 


What is the expected output? What do you see instead?
Talkback is reading strange numbers which aren't even part of the real number 
often starting with +- and are in the range of billions. not even the country 
prefix matches. This applies when the focus is moved either by touch or by the 
hardware keyboard arrows to the corresponding number. The name of the number, 
e.g. private or mobile seems to be correct however. The number is displayed on 
screen correctly. This behaviour does not occur in the contacts app. However 
talkback just reads the beginning of landline numbers.

What version of the product are you using? On what operating system?
talkback-v3.1.4-beta6 on Cyanogen mod 10 M2 (fxp142) jellybean on Sony XPeria 
mini pro

```

Original issue reported on code.google.com by `brandl...@googlemail.com` on 17 Oct 2012 at 8:27

**Comment by GoogleCodeExporter** [comment]
```
This is an issue with the Google TTS and cannot be fixed in TalkBack.
```

Original comment by `alanv@google.com` on 21 Nov 2012 at 1:38
- Changed state: **WontFix**
